### PR TITLE
Suppress plugins for global options and sub-commands.. Closes #469

### LINKED ISSUE #469: [BUG]: running `devproxy --version` with `DevToolsPlugin` configured start the browser
### Description

running `devproxy --version` with `DevToolsPlugin` configured start the browser but I think should just present the version

### Expected behaviour

running `devproxy --version` with `DevToolsPlugin` should just present the version without starting the browser 

### Actual behaviour

starts the browser and presents this 
image

### Steps to reproduce

1. install Dev Proxy
2. Configure DevToolsPlugin following the guide for `Edge` as browser
3. run `devproxy --version`

### Dev Proxy Version

0.13.0

### Operating system (environment)

Windows

### Shell

PowerShell

### Configuration file

{
  "plugins": [
    {
      "name": "DevToolsPlugin",
      "enabled": false,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "devTools"
    },
    {
      "name": "LatencyPlugin",
      "enabled": false,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "latencyPlugin"
    },
    {
      "name": "RetryAfterPlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll"
    },
    {
      "name": "GraphSelectGuidancePlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "urlsToWatch": [
        "https://graph.microsoft.com/v1.0/*",
        "https://graph.microsoft.com/beta/*",
        "https://graph.microsoft.us/v1.0/*",
        "https://graph.microsoft.us/beta/*",
        "https://dod-graph.microsoft.us/v1.0/*",
        "https://dod-graph.microsoft.us/beta/*",
        "https://microsoftgraph.chinacloudapi.cn/v1.0/*",
        "https://microsoftgraph.chinacloudapi.cn/beta/*"
      ]
    },
    {
      "name": "ODSPSearchGuidancePlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "urlsToWatch": [
        "https://graph.microsoft.com/v1.0/*",
        "https://graph.microsoft.com/beta/*",
        "https://graph.microsoft.us/v1.0/*",
        "https://graph.microsoft.us/beta/*",
        "https://dod-graph.microsoft.us/v1.0/*",
        "https://dod-graph.microsoft.us/beta/*",
        "https://microsoftgraph.chinacloudapi.cn/v1.0/*",
        "https://microsoftgraph.chinacloudapi.cn/beta/*"
      ]
    },
    {
      "name": "GraphBetaSupportGuidancePlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "urlsToWatch": [
        "https://graph.microsoft.com/beta/*",
        "https://graph.microsoft.us/beta/*",
        "https://dod-graph.microsoft.us/beta/*",
        "https://microsoftgraph.chinacloudapi.cn/beta/*"
      ]
    },
    {
      "name": "GraphSdkGuidancePlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "urlsToWatch": [
        "https://graph.microsoft.com/v1.0/*",
        "https://graph.microsoft.com/beta/*",
        "https://graph.microsoft.us/v1.0/*",
        "https://graph.microsoft.us/beta/*",
        "https://dod-graph.microsoft.us/v1.0/*",
        "https://dod-graph.microsoft.us/beta/*",
        "https://microsoftgraph.chinacloudapi.cn/v1.0/*",
        "https://microsoftgraph.chinacloudapi.cn/beta/*"
      ]
    },
    {
      "name": "ODataPagingGuidancePlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll"
    },
    {
      "name": "GraphClientRequestIdGuidancePlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "urlsToWatch": [
        "https://graph.microsoft.com/v1.0/*",
        "https://graph.microsoft.com/beta/*",
        "https://graph.microsoft.us/v1.0/*",
        "https://graph.microsoft.us/beta/*",
        "https://dod-graph.microsoft.us/v1.0/*",
        "https://dod-graph.microsoft.us/beta/*",
        "https://microsoftgraph.chinacloudapi.cn/v1.0/*",
        "https://microsoftgraph.chinacloudapi.cn/beta/*"
      ]
    },
    {
      "name": "CachingGuidancePlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "cachingGuidance"
    },
    {
      "name": "RateLimitingPlugin",
      "enabled": false,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "rateLimiting"
    },
    {
      "name": "MockResponsePlugin",
      "enabled": false,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "mocksPlugin"
    },
    {
      "name": "GraphMockResponsePlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "mocksPlugin"
    },
    {
      "name": "GraphRandomErrorPlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "graphRandomErrorsPlugin"
    },
    {
      "name": "ExecutionSummaryPlugin",
      "enabled": false,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "executionSummaryPlugin"
    },
    {
      "name": "MinimalPermissionsPlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "minimalPermissionsPlugin"
    },
    {
      "name": "MinimalPermissionsGuidancePlugin",
      "enabled": false,
      "pluginPath": "plugins\\dev-proxy-plugins.dll"
    },
    {
      "name": "MockGeneratorPlugin",
      "enabled": false,
      "pluginPath": "plugins\\dev-proxy-plugins.dll"
    },
    {
      "name": "DevToolsPlugin",
      "enabled": true,
      "pluginPath": "plugins\\dev-proxy-plugins.dll",
      "configSection": "devTools"
    }
  ],
  "devTools": {
    "preferredBrowser": "Edge"
  },
  "urlsToWatch": [
    "https://graph.microsoft.com/v1.0/*",
    "https://graph.microsoft.com/beta/*",
    "https://graph.microsoft.us/v1.0/*",
    "https://graph.microsoft.us/beta/*",
    "https://dod-graph.microsoft.us/v1.0/*",
    "https://dod-graph.microsoft.us/beta/*",
    "https://microsoftgraph.chinacloudapi.cn/v1.0/*",
    "https://microsoftgraph.chinacloudapi.cn/beta/*",
    "https://*.sharepoint.*/*_api/*",
    "https://*.sharepoint.*/*_vti_bin/*",
    "https://*.sharepoint-df.*/*_api/*",
    "https://*.sharepoint-df.*/*_vti_bin/*"
  ],
  "mocksPlugin": {
    "mocksFile": "responses.json"
  },
  "graphRandomErrorsPlugin": {
    "allowedErrors": [ 429, 500, 502, 503, 504, 507 ]
  },
  "executionSummaryPlugin": {
    "groupBy": "url"
  },
  "minimalPermissionsPlugin": {
    "type": "delegated"
  },
  "cachingGuidance": {
    "cacheThresholdSeconds": 5
  },
  "latencyPlugin": {
    "minMs": 200,
    "maxMs": 10000
  },
  "rateLimiting": {
    "costPerRequest": 2,
    "rateLimit": 120,
    "retryAfterSeconds": 5
  },
  "rate": 50,
  "labelMode": "text",
  "logLevel": "info"
}

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by garrytrinder**
Closes #469

### CODE REVIEW

**Comment by garrytrinder** [comment]
Subcommands are also effected, so marking PR as draft whilst working on fix.

File: dev-proxy/devproxyrc.json [hunk 1]
```diff
@@ -11,6 +11,12 @@
       "name": "RetryAfterPlugin",
       "enabled": true,
       "pluginPath": "plugins\\dev-proxy-plugins.dll"
+    },
+    {
+      "name": "DevToolsPlugin",
```
Comment (waldekmastykarz) [inline dev-proxy/devproxyrc.json:16]: Is this change intentional?

**Review by waldekmastykarz — APPROVED** [review]
